docs/common_tasks.md

```diff
@@ -41,3 +41,11 @@ To add a new repository, the following steps are needed:
 
 For more information on adding a new GitHub repository, see
 [Adding a GitHub repository](submitting_data.md#adding-a-github-repository).
+
+## Debugging Tools
+
+You can use the Python Debugger ([pdb](https://docs.python.org/3.7/library/pdb.html)) in a docker container.
```

Comment:
Please capitalize Docker and Treeherder (next line).

Reply:
Will do.